### DISCUSSION TITLE
Integration with SugarCRM

### DISCUSSION THREAD

**Discussion by terryupton** [solspace/craft-freeform#1591] [Feature Requests]
Hi @kjmartens,

Are there any plans to integrate with SugarCRM? 
https://www.sugarcrm.com/uk/

I have a client that uses this CRM system and we are potentially about to embark on a rebuild of their Craft website and they have requested that the webforms integrate with their CRM (SugarCRM). Currently Freeform is my go to for forms, so I wanted to see if there were any plans to offer this integration?

Thanks
Terry

**Comment by kjmartens** [comment]
Thanks for your feedback @terryupton.

We will review adding support for this very soon. 🙂

**Comment by kjmartens** [comment]
This is now available in Freeform 5.13+ 🙂
